scheduler/build.gradle

```diff
@@ -53,6 +53,7 @@ dependencies {
   compileOnly project(':constraints')
   compileOnly project(':contrib')
   compileOnly project(':examples:banananation')
+  compileOnly project(':examples:foo-missionmodel')
```

Comment:
Are `banananation` and `foo-missionmodel` being used outside of `src/test/...`? If not, I'd remove line 55 & 56 here. From a quick grep it looks like they're only expected to be used within `src/test/...`.